mkosi

```diff
@@ -2795,13 +2823,25 @@ def reuse_cache_tree(args, workspace, run_build_script, for_cache, cached):
 
     return True
 
+def make_output_dir(args):
+    """Create the output directory if set and not existing yet"""
+    if args.output_dir is None:
+        return
+
+    try:
+        os.mkdir(args.output_dir, 0o755)
```

Comment:
Maybe add a comment here that `mkdirs` is not appropriate because we only want to create the final component.